### PR TITLE
Fix for `UNKNOWN DATABASE` exception

### PR DESCRIPTION
**Pull request by IlyaTsoi**
We get a `UNKNOWN DATABASE` exception because a connection string uses target DB.
`runQuery()` tries to execute `CREATE DATABASE` statement within an absent context. We can use multistatement as a workaround to execute `CREATE DATABASE` query within a `system` DB context

### CODE REVIEW

File: sink-connector/src/test/java/com/altinity/clickhouse/sink/connector/db/operations/ClickHouseCreateDatabaseTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+public class ClickHouseCreateDatabaseTest {
+
+    static DbWriter dbWriter;
+    static DbWriter maintenanceDbWriter;
+    static String dbName;
+
+    @Container
+    private ClickHouseContainer clickHouseContainer = new ClickHouseContainer("clickhouse/clickhouse-server:latest")
+            .withInitScript("./init_clickhouse.sql");;
```
Comment (subkanthi) [inline sink-connector/src/test/java/com/altinity/clickhouse/sink/connector/db/operations/ClickHouseCreateDatabaseTest.java]: extra semicolon

**Comment by subkanthi** [comment]
@IlyaTsoi can u please check why the test u added is failing, please pull latest from develop.

**Comment by IlyaTsoi** [comment]
> @IlyaTsoi can u please check why the test u added is failing, please pull latest from develop.

I did try) But I can't reproduce this error in my environment, test passes successfully.

**Comment by subkanthi** [comment]
> > @IlyaTsoi can u please check why the test u added is failing, please pull latest from develop.
> 
> I did try) But I can't reproduce this error in my environment, test passes successfully.

`Error connectingjava.sql.SQLException: Connection refused (Connection refused), server ClickHouseNode [uri=http://localhost:8123/test, options={client_name=Test_1}]@630096182`

instead of localhost and 8123, you should get the host and port from clickHouseContainer. 

```
        String hostName = clickHouseContainer.getHost();
        Integer port = clickHouseContainer.getFirstMappedPort();
```

**Review by subkanthi — APPROVED** [review]

**Pull request merged**